lib/helper/Appium.js

```diff
@@ -276,6 +276,26 @@ class Appium extends WebdriverIO {
     return recorder.promise();
   }
 
+  /**
+   * Execute code only in Web mode.
+   *
+   * ```js
+   * I.runInWeb(() => {
+   *    I.waitForElement('#data');
+   *    I.seeInCurrentUrl('/data');
+   * });
+   * ```
+   *
+   * @param {*} fn
+   */
+  runInWeb(fn) {
+    if (!this.isWeb) return;
+    recorder.session.start('Web-only actions');
+    fn();
+    recorder.add('restore from Web session', () => recorder.session.restore());
```

Comment:
I think, we should have tests for this cool features like different locators for different platforms, execution only on specified platform. This is killer feature and we have to be sure that it's works after each update